tools/config-generator/templates/prow_presubmit_job.yaml

```diff
@@ -3,6 +3,7 @@
     [[indent_section 6 "labels" .Base.Labels]]
     context: [[.PresubmitPullJobName]]
     always_run: [[.Base.AlwaysRun]]
```

Comment:
Is there a conflict if `always_run: true` and `run_if_changed` is specified? If not, which one overrides the other?

Reply:
This is no conflict, but `always_run: true` has a higher priority, that means:
1. if `always_run: true`, the Prow job will always be run regardless the value of `run_if_changed`
2. if no `always_run:true`, the Prow job will be run if the changed files match with the value of `run_if_changed`.

Details can be found in https://github.com/kubernetes/test-infra/blob/master/prow/config/jobs.go#L347-L363

Reply:
kk, thanks for digging it out